toil-appliance/test/test.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+mkdir ./tempDir
+
+LOCAL_PATH=$(cd ./tempDir; pwd)
+LEADER_PATH=/test/data
+UUID=$(python -c 'import uuid; print uuid.uuid4()')
+LEADER_IP=172.17.0.2
```

Comment:
Try passing `--network=container:$LEADER` to the worker container. But that still relies a little to heavily on the defaults for my taste. Can you open a ticket to revisit this?